### PR TITLE
add array2gif

### PR DESCRIPTION
**Pull request by tanyaschlusser**
Add 'array2gif' recipe -- makes animated/static gifs from NumPy arrays. Thank you!

### CODE REVIEW

File: recipes/array2gif/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - numpy
```
Comment (synapticarbors) [inline recipes/array2gif/meta.yaml]: I think you can drop `numpy` as a `build` dependency.

File: recipes/array2gif/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - numpy
```
Comment (synapticarbors) [inline recipes/array2gif/meta.yaml]: You don't need to test the import of numpy.

File: recipes/array2gif/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package:
```
Comment (synapticarbors) [inline recipes/array2gif/meta.yaml]: You might consider using jinja templating for the `package` and `source` sections as shown in the example:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L21

File: recipes/array2gif/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{% set name = "array2gif" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "d8c9ca4b7656f4016c578e6b0f87daf47114f5456db2fceece896be5d173e9f0" %}
+
+package:
+  name: {{ name|lower ||
```
Comment (synapticarbors) [inline recipes/array2gif/meta.yaml]: Looks like a typo here with `||` instead of `}}'

**Comment by tanyaschlusser** [comment]
Sorry for the trouble -- thank you!

**Comment by synapticarbors** [comment]
I apologize, but you actually need `numpy` in the `build` requirements section. I overlooked that the library was being imported in the `setup.py`, which as a side effect tries to import `numpy`. I think after that, everything should be ready to merge.

**Comment by synapticarbors** [comment]
@tanyaschlusser -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**